packages/cli/src/commands/generate/scaffold/scaffold.js

```diff
@@ -487,6 +529,18 @@ const componentFiles = async (
     })
   )
 
+  const formattersImports = columns
+    .map((column) => column.displayFunction)
+    .sort()
+    .filter((v, i, a) => a.indexOf(v) === i)
+    .join(', ')
+
+  const listFormattersImports = columns
+    .map((column) => column.listDisplayFunction)
+    .sort()
+    .filter((v, i, a) => a.indexOf(v) === i)
```

Comment:
using variable names here would be super helpful :)

Reply:
ugh. 100% agree!